Update possible values for `Credential.type` (#32117)
mdn/content: files/en-us/web/api/credential/id/index.md

{{APIRef("Credential Management API")}}{{SecureContext_Header}}

> The **`id`** read-only property of the {{domxref("Credential")}} interface returns a string containing the credential's identifier. This might be any one of a GUID, username, or email address.

Reviewer: "This might be any one of GUID, username ..." restricts the list to the current types. I can't find a clear list in the spec, and in some ways it is better not to fix the list so that if new types of credential appear this cannot become wrong.

```suggestion
The **`id`** read-only property of the {{domxref("Credential")}} interface returns a string containing the credential's identifier. This might be a GUID, username, or email address, or some other value, depending on the type of credential.
```

Author: Yeah, because it relates with four standard (maybe more in the future), it seems a good fix
